.github/workflows/ci.yml

```diff
@@ -91,6 +91,10 @@ jobs:
       with:
         path: ./src/github.com/linuxkit/linuxkit
 
+    - name: Set up binfmt
+      # Only register arm64 as we are on amd64 already. s390x is not reliable
+      run: docker run --privileged --rm tonistiigi/binfmt --install arm64
```

Comment:
why arent we using our own binfmt package?

Reply:
because the buildx instructions said to use tonis's one :joy: i can change it